### PR TITLE
Add new resources spoke_gateway and transit_gateway

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

**Comment by arvindsree** [comment]
Update test-infra folder as well with the variables

**Comment by zhiqiangjin** [comment]
Will do another PR to update test-infra.

File: README_accep_test.md [hunk 1]
```diff
@@ -32,20 +32,27 @@ Passing an environment value of "yes" to the skip parameter allows you to skip t
...
+|                                      | SKIP_SPOKE_AWS               |         + AWS_VPC_ID, AWS_REGION, AWS_SUBNET, AWS_GW_SIZE (optional)  |
 |                                      | SKIP_SPOKE_GCP               |         + GCP_VPC_ID, GCP_ZONE, GCP_SUBNET, GCP_GW_SIZE (optional)    |
 |                                      | SKIP_SPOKE_ARM               |         + ARM_VNET_ID, ARM_REGION, ARM_SUBNET, ARM_GW_SIZE            |
 | aviatrix_trans_peer                  | SKIP_TRANS_PEER              | aviatrix_tunnel                                                       |
+| aviatrix_transit_gateway             | SKIP_TRANSIT_GATEWAY         | aviatrix_gateway                                                      |
+|                                      | SKIP_TRANSIT_GATEWAY_AWS     | aviatrix_gateway in AWS                                               |
+|                                      | SKIP_TRANSIT_GATEWAY_ARM     | aviatrix_gateway in ARM                                               |
 | aviatrix_transit_vpc                 | SKIP_TRANSIT                 | aviatrix_gateway                                                      |
```
Comment (arvindsree) [inline README_accep_test.md]: Delete these lines

File: README_accep_test.md [hunk 2]
```diff
@@ -32,20 +32,27 @@ Passing an environment value of "yes" to the skip parameter allows you to skip t
...
 |                                      | SKIP_GCP_GATEWAY             |         + GCP_VPC_ID, GCP_ZONE, GCP_SUBNET, GCP_GW_SIZE (optional)    |
 |                                      | SKIP_ARM_GATEWAY             |         + ARM_VNET_ID, ARM_REGION, ARM_SUBNET, ARM_GW_SIZE            |
 | aviatrix_site2cloud                  | SKIP_S2C                     | aviatrix_gateway                                                      |
+| aviatrix_spoke_gateway               | SKIP_SPOKE_GATEWAY           | aviatrix_gateway                                                      |
+|                                      | SKIP_SPOKE_GATEWAY_AWS       |         + AWS_VPC_ID, AWS_REGION, AWS_SUBNET, AWS_GW_SIZE (optional)  |
+|                                      | SKIP_SPOKE_GATEWAY_GCP       |         + GCP_VPC_ID, GCP_ZONE, GCP_SUBNET, GCP_GW_SIZE (optional)    |
+|                                      | SKIP_SPOKE_GATEWAY_ARM       |         + ARM_VNET_ID, ARM_REGION, ARM_SUBNET, ARM_GW_SIZE            |
 | aviatrix_spoke_vpc                   | SKIP_SPOKE                   | aviatrix_gateway                                                      |
```
Comment (arvindsree) [inline README_accep_test.md]: Same here

**Pull request merged**